tests/terranix-doc-json-tests/01.nix.output

```diff
@@ -1 +1 @@
-{"testing-terranix.enable":{"declarations":[{"channelPath":"/01.nix","path":"01.nix","url":"http://example.com/01.nix"}],"default":false,"description":"Whether to enable enable testing-terranix.","example":true,"loc":["testing-terranix","enable"],"readOnly":false,"type":"boolean"}}
+{"testing-terranix.enable":{"declarations":[{"channelPath":"/01.nix","path":"01.nix","url":"http://example.com/01.nix"}],"default":{"_type":"literalExpression","text":"false"},"description":"Whether to enable enable testing-terranix.","example":{"_type":"literalExpression","text":"true"},"loc":["testing-terranix","enable"],"readOnly":false,"type":"boolean"}}
```

Comment:
These `_type` parameters are not that nice, but I guess they don't generate problems.
I guess we have to fix the doc-json function with this update. What do you think @mdarocha ?

Reply:
If the behavior of Terraform 1.5 has changed, isn't this update necessary to maintain compatibility, even if it is a step backwards?

Reply:
This has nothing to do with terraform. Terraform is actually not part of this project. (only an old version appears in the dev shell).

The problem is with the changes in the nixos module renderer that changed because of nix flake update. I currently have no time to find out how to fix this, sorry. But as soon as I have time I would tackle this problem.